koku/api/report/queries.py

```diff
@@ -1075,6 +1077,10 @@ def _ranked_list(self, data_list, ranks):
         # Cross join ranks and days to get each field/rank for every day in th query
         ranks_by_day = rank_data_frame.merge(day_data_frame, how="cross")
 
+        # add the ranking column if it still exists in both dataframes
+        if rank_field and rank_field[0] in ranks_by_day.columns.intersection(data_frame.columns):
```

Comment:
Is `rank_field` expected to be a single item list passed into this method?

Reply:
yesf

Reply:
On second look, it does appear that rank_field may contain more than 1 entry. I'll update this logic here to check for all possible entries.

Reply:
Here is a query which results in a `rank_field` containing multiple fields:
```
api/cost-management/v1/reports/openshift/compute/?filter[limit]=50&filter[offset]=0&filter[resolution]=monthly&filter[time_scope_units]=month&filter[time_scope_value]=-2&group_by[project]=*&order_by[delta]=desc&delta=request__limit
```